settings.html

```diff
@@ -6,17 +6,18 @@
 <br>
 <section id="options" class="centered topSpace">
 
-    <label for="use24HourTime" >Use 24-hour Time? </label>
-    <input type="checkbox" id="use24HourTime" onchange="updateSettings()" />
+    <label>Use 24-hour Time? 
+        <input type="checkbox" id="use24HourTime" onchange="updateSettings()" />
+    </label>
 
+    <em>Settings are automatically saved</em>
 </section>
 
-<p class="centered topSpace">Settings are automatically saved</p>
 
 <section id="credits" class="centered topSpace">
 
     <a class=" topSpace bottomSpace" href="https://github.com/MoralCode/ClassClock">View on GitHub</a>
-    <a class=" topSpace" href="https://github.com/MoralCode/ClassClock/issues/new/choose">Submit Feedback</a><br>
```

Comment:
We're no longer accepting feedback?

Reply:
we are, i just thought Github wasnt the most user-friendly place to use. we now have twitter, instagram and slack as potential support options (until we make an FAQ and/or support page)

Reply:
Ah - true. Speaking of, did you see the @WeAreLO page mentioned us?